zerver/lib/events.py

```diff
@@ -356,6 +356,7 @@ def fetch_initial_state_data(
             state["demo_organization_scheduled_deletion_date"] = datetime_to_timestamp(
                 realm.demo_organization_scheduled_deletion_date
             )
+        state["realm_date_created"] = datetime_to_timestamp(realm.date_created)
```

Comment:
This is an API change that needs to be documented in the API changelog and API documentation. The "new application feature" tutorial and linked documentation on API updates should guide your through what needs to happen there.